tests/test_dmlrunner.py

```diff
@@ -19,6 +23,42 @@ def config_manager():
     )
     return config_manager
 
+
+@pytest.fixture
+def sample_transform_function():
+    return lambda x: x.drop_duplicates()
+
+@pytest.fixture
+def mnist_filepath():
+    return 'tests/artifacts/runner_scheduler/mnist'
+
+@pytest.fixture
+def small_filepath():
+    return 'tests/artifacts/runner_scheduler/test'
+
+def accurate_transform_test(dsm, expected_test_transformed, split):
+    """
+    Check that the data in this folder is the result of calling the transform function on
+    each csv in the raw data filepath
+    """
+    transform_dsm = dsm.get_transformed_data()
+    keys = list(transform_dsm.keys())
```

Comment:
doesn't look like this variable is used anywhere?

Reply:
Not using this function, will delete.